docs/src/main/asciidoc/maven-tooling.adoc

```diff
@@ -652,8 +652,8 @@ Uber-Jar's final name is configurable via a Maven's build settings `finalName` o
 
 ==== Uber-Jar file name suffix
 
-By default the generated uber JAR file name will have the `-runner` suffix, unless it was overridden by configuring a custom one with `quarkus.package.runner-suffix` configuration option.
-If the runner suffix is not desired, it can be disabled by setting `quarkus.package.jar.add-runner-suffix` configuration option to `false`, in which case the uber JAR will replace the original JAR
+By default the generated uber JAR file name will have the `-runner` suffix, unless it was overridden by configuring a custom one with `quarkus.package.jar.runner-suffix` configuration option.
+If the runner suffix is not desired, it can be disabled by setting `quarkus.package.add-runner-suffix` configuration option to `false`, in which case the uber JAR will replace the original JAR
```

Comment:
Can you confirm the second change shouldn't be applied?

Reply:
After investigating the issue raised in Quarkus issue #45204, I found that this behavior changed in Quarkus 3.10+. Therefore, I initially added a fix to remove the parameters. However, after becoming aware of the change in the current release, I re-added the parameters to ensure compatibility.

Reply:
What @gsmet is asking is whether `quarkus.package.add-runner-suffix` is correct, or whether it should be `quarkus.package.jar.add-runner-suffix`.

Reply:
Version 3.10+ should be quarkus.package.jar.add-runner-suffix

Reply:
Thanks. In that case please update the docs in this PR.

Reply:
Updated.